src/e2e/resources/simple-temple-expected/simple-temple-test-user/dao/dao.go

```diff
@@ -191,3 +246,88 @@ func (dao *DAO) IdentifySimpleTempleTestUser(input IdentifySimpleTempleTestUserI
 
 	return &simpleTempleTestUser, nil
 }
+
+// ListFred returns a list containing every fred in the datastore
+func (dao *DAO) ListFred(input ListFredInput) (*[]Fred, error) {
+	rows, err := executeQueryWithRowResponses(dao.DB, "SELECT id, created_by, field, friend, image FROM fred WHERE parent_id = $1;", input.ParentID)
+	if err != nil {
+		return nil, err
+	}
+
+	fredList := make([]Fred, 0)
+	for rows.Next() {
+		var fred Fred
+		err = rows.Scan(&fred.ID, &fred.Field, &fred.Friend, &fred.Image)
+		if err != nil {
+			return nil, err
+		}
+		fredList = append(fredList, fred)
+	}
+	err = rows.Err()
+	if err != nil {
+		return nil, err
+	}
+
+	return &fredList, nil
+}
+
+// CreateFred creates a new fred in the datastore, returning the newly created fred
+func (dao *DAO) CreateFred(input CreateFredInput) (*Fred, error) {
+	row := executeQueryWithRowResponse(dao.DB, "INSERT INTO fred (id, created_by, field, friend, image) VALUES ($1, $2, $3, $4, $5) RETURNING id, created_by, field, friend, image;", input.ID, input.Field, input.Friend, input.Image)
```

Comment:
Where do we set the `parent_id`?

Reply:
should do here, now that’s been merged in